redis/README.md

```diff
@@ -31,3 +37,31 @@ OPSLEVEL_APP_TIER="tier_4" # Tier of your Redis in OpsLevel- see https://wiki.uw
 REDIS_SECRET_NAME="redis" # Name of the secret with your Redis password. See secret created in directory `example`.
 REDIS_REPLICA_COUNT=1 # Amount of read-only replicas 
 ```
+
+## Usage
+In order to manually access the database:
+1. [Install redis-cli](https://redis.io/docs/install/install-redis/) (it comes together with server installation)
+2. port-forward to Redis service running in the cluster:
+    ```bash
+    kubectl port-forward svc/redis-shared-master 6379
+    ```
+3. Connect redis-cli to service:
+    ```bash
+    redis-cli -u redis://localhost:6379 -a "password" 
```

Comment:
I think this can be done in a single `exec` (is that nicer?)

```suggestion
    kubectl exec --tty --stdin=true  svc/redis-shared-master -- redis-cli -u redis://localhost:6379 -a "password" 
```

Reply:
Nice idea, [applied it](https://github.com/utilitywarehouse/shared-kustomize-bases/pull/11/commits/f9a717c57851af4bd723ae23dcde91a701b82a30)!